iModelCore/ECDb/ECDb/ConcurrentQueryManagerImpl.h

```diff
@@ -208,27 +208,30 @@ struct RunnableRequestBase {
         std::atomic_bool m_cancelled;
         uint32_t m_executorId;
         uint32_t m_connId;
+        bool m_interrupted;

```

Comment:
Not sure about threading scenario here - does this need to be atomic?

Reply:
The variable is only accessed by a single thread through a monitor call. Since no other thread accesses this value, declaring it as atomic may not be necessary. However, for the sake of code readability and to future-proof against potential access by other threads, I will declare it as atomic.